nemo_skills/pipeline/train.py

```diff
@@ -251,7 +259,7 @@ def train(
     average_steps: str = typer.Option(
         'all',
         help="List of commas separated checkpoint steps to average. E.g 1000,5000. "
-        "If None, will skip prepare eval stage.",
+        "If None, will save the last checkpoint.",
```

Comment:
Let's make it a separate parameter. Or perhaps use that when average_steps = 'last'. I think it's still useful to be able to skip prepare eval stage completely if someone want to have their own way of handling checkpoints

Reply:
thanks for checking this! wasn't sure which way we wanted to go, agree with this!